00-init.tf

```diff
@@ -5,5 +5,9 @@ terraform {
       source                = "hashicorp/azurerm"
       configuration_aliases = [azurerm.hmcts-control, azurerm.acr, azurerm.global_acr]
     }
+     azapi = {
```

Comment:
formatting looks off have you run `terraform fmt`?

Reply:
Have corrected format